resources/setups/qualification/minimal_device_module_config_template.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
this filename is horrific so really need something more structured.  It's hard for me to parse this change out from the other ones.... (although it's all "config changes" it looks like there's a bunch of semantically different changes?)  Is there a doc or something that outlines what the desired structure is?

Reply:
Thanks @grafnu for having a look!
Yes, I agree it is a long and convoluted name, but it was meant to be renamed when copied.
I agree that renaming contextually would work best.
[Here is a proposed naming document ](https://docs.google.com/document/d/1CziZjgm3UFzKdA7Bd54sAQpvgm6Oac5hbgjemS4Qd4A/edit) for those files, let's discuss there as you've suggested. Thanks!

Reply:
I've updated the names of the files inside the `resources/setups` directory and moved `device_types` inside a `library` directory as discussed in the naming document.